README.md

```diff
@@ -62,6 +62,13 @@ Engine](https://docs.docker.com/installation/)).
 and/or logs directory, run as `sudo -E docker-compose up` to pass in environment variables). The
 first run will take longer since the Docker images will be built and then run.
 
+### Using Vagrant
+This method runs Docker Engine inside a Vagrant VM.
+
+1. In the project directory, run `vagrant up`. This will also build the Docker images inside the
+VM and set all environment variables.
+1. `vagrant ssh` into the VM and then start the service with `sudo -E qata start`.
```

Comment:
@mDibyo I can confirm that everything worked correctly and was able to access the dashboard on Mac. Great job!
Would love to start showing this on a monitor in our office